Dockerfile.assisted-iso-create

```diff
@@ -0,0 +1,22 @@
+#This image contains the RHCOS livecd.iso
+FROM quay.io/ocpmetal/livecd-iso:rhcos-livecd AS livecd-build
+RUN chmod 666 /root/image/livecd.iso
+
+FROM quay.io/coreos/coreos-installer:release AS installer-image
+
+FROM python:3-slim
```

Comment:
Do we still need python? Or can we base this on UBI?

Reply:
i will have to check. the main reason for using python3-slim image is because it has /lib64/ld-linux-x86-64.so.2 program interpreter  installed, which is needed by coreos-installer. If UBI has it, then i guess we can use it as well

Reply:
Looks like ubi-minimal has it.

```
$ podman run -it --rm registry.access.redhat.com/ubi8/ubi-minimal ls -l /lib64/ld-linux-x86-64.so.2
lrwxrwxrwx. 1 root root 10 Jan 16  2020 /lib64/ld-linux-x86-64.so.2 -> ld-2.28.so
```

Reply:
switch to ubi-minimal